website_sale_attribute_filter_category/controllers/main.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2019 Tecnativa - Sergio Teruel
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.addons.website_sale.controllers.main import WebsiteSale
```

Comment:
Is this compatible with other modules overwriting same class?

Reply:
This is a controller, I only add data to qcontext.

Reply:
Yeah, mine is a general question about controllers. In my understanding, as this uses Python inheritance, you won't have all the inheritance chain through this. Am I right?

Reply:
Yes